test/unit/math/prim/prob/normal_ccdf_log_test.cpp

```diff
@@ -12,3 +12,38 @@ TEST(ProbNormal, ccdf_log_matches_lccdf) {
       (stan::math::normal_lccdf<double, double, double>(y, mu, sigma)),
       (stan::math::normal_ccdf_log<double, double, double>(y, mu, sigma)));
 }
+
+TEST(ProbNormal, lccdf_tail) {
+  using stan::math::normal_lccdf;
+
+  EXPECT_FLOAT_EQ(1, -6.661338147750941214694e-16 / normal_lccdf(-8.0, 0, 1));
```

Comment:
Is there a reason the direct comparison isn't done? Meaning...
`EXPECT_FLOAT_EQ(-6.661338147750941214694e-16, normal_lccdf(-8.0, 0, 1));`

If I'm to guess, it's because of the amount of precision near 1 instead of 0?

Reply:
I thought so, but using direct comparisons all tests pass as well (on Linux, clang++), so I'll switch to using those.